torch/nn/modules/container.py

```diff
@@ -480,6 +480,12 @@ def extra_repr(self) -> str:
     def __call__(self, input):
         raise RuntimeError('ParameterList should not be called.')
 
+    def _replicate_for_data_parallel(self)
+        warnings.warn("nn.ParameterList is being used with DataParallel but this is not "
+                      "officieally supported. This list will appear empty for the models "
```

Comment:
officieally -> officially 

is it unofficially supported or should we just say it's not supported?

Reply:
Well, I don't want to make this an error to not break code. But yes "not supported" sounds better.